docs/source/development/traffic_ops_api/v12/topology.rst

```diff
@@ -23,11 +23,109 @@ Snapshot CRConfig
 /api/1.2/snapshot/{:cdn_name}
```

Comment:
Have we always used "{:cdn_name}" for the parameter? I thought we used ":cdn_name"